.github/workflows/sage-test.yml

```diff
@@ -0,0 +1,149 @@
+            fi
+            # Test that terminus works...
+            terminus --version
+      - name: Validate Pantheon Host Key
+        run: |
+          echo "Host *.drush.in HostKeyAlgorithms +ssh-rsa" >> ~/.ssh/config
+          echo "Host *.drush.in PubkeyAcceptedKeyTypes +ssh-rsa" >> ~/.ssh/config
+          echo "StrictHostKeyChecking no" >> ~/.ssh/config
+      - name: Log into Terminus and Check for updates
+        run: |
+          terminus auth:login --machine-token=${{ secrets.TERMINUS_TOKEN }}
+          terminus upstream:updates:apply wpcm-sage-install-tests.dev --accept-upstream
+      - name: Clone site and create multidev
+        run: |
+          echo "Cloning site..."
+          terminus local:clone wpcm-sage-install-tests
+          cd ~/pantheon-local-copies/wpcm-sage-install-tests
+          if [[ -n $(git status --porcelain) ]]; then
+            echo "❌ Local clone is dirty. Exiting..."
+            exit 1
+          fi
+          # If sage-test exists, delete it.
+          if [[ -d "web/app/themes/sage-test" ]]; then
+            echo "Deleting existing sage-test..."
+            rm -rf web/app/themes/sage-test
+          fi
+          terminus multidev:create wpcm-sage-install-tests.dev $multidev_name
+          echo "Checking out multidev..."
+          git fetch --all
+          git checkout $multidev_name
+      - name: Copy latest repository changes
+        run: |
+          # Sync the files from checked-out repo to pantheon-local-copies, excluding the .git folder
+          rsync -av --exclude='.git/' ${{ github.workspace }}/ ~/pantheon-local-copies/wpcm-sage-install-tests/
+          # Navigate to Pantheon local copies directory
+          cd ~/pantheon-local-copies/wpcm-sage-install-tests/
+          git config --global user.email "bot@getpantheon.com"
+          git config --global user.name "Pantheon TestBot"
+          # Add, commit and push
+          git add .
+          git commit -m "Sync latest changes to test environment"
+          git push origin $multidev_name
+      - name: Install Composer Dependencies
+        run: |
+          cd ~/pantheon-local-copies/wpcm-sage-install-tests
+          composer install
+      - name: Run Sage Install Script
+        run: |
+          export SAGENAME=sage-test
+          export SITENAME=wpcm-sage-install-tests
+          export CI=1
+          export SITEENV=$multidev_name
+          export PHPVERSION=${{ matrix.php-version }}
+          cd ~/pantheon-local-copies/$SITENAME
+          composer install-sage || exit_code=$?
+          if [[ -n "${exit_code}" ]]; then
+            echo "❌ Sage Install Script Failed with exit code ${exit_code}"
+            exit $exit_code
+          else
+            echo "✅ Sage Install Script passed!"
+            exit 0;
+          fi
+      - name: Delete multidev
+        if: always()
+        run: terminus multidev:delete --delete-branch --yes wpcm-sage-install-tests.$multidev_name
```

Comment:
Is multidev env not useful to review in case of failed tests? Seems should should only run on success, and otherwise have validation _in front_ of the CI that ensures there are enough environments.

Reply:
I don't know that there's anything the multidev would tell you (that you couldn't get from workflow logs) that you wouldn't see in the output of the automated run. What's more, because the multidev environment is named based on the PR #, it will always create a multidev named `${os}-${php_version}-${pr_num}` and then we'd need to do all sorts of cleanup on the environment for subsequent runs on the same PR.